api-server/source/patent_crawler.py

```diff
@@ -0,0 +1,13 @@
+from dynamodb_patentdata import save_patent_datas
+from get_data import get_patent_datas, store_only_ids
+from get_account import get_patent_ids
+
+def handler(event, context):
+    patent_id= get_patent_ids()
+    store_patent_id=store_only_ids(patent_id)
```

Comment:
```suggestion
    stored_patent_id=store_only_ids(patent_id)
```
store_patent_id sounds like it is a function